Adding CLI changes, meshId is now assigned automatically (#20)
AdobeDocs/graphql-mesh-gateway: src/pages/index.md

---
title: Overview - API Mesh for Adobe Developer App Builder

> description: This is the overview page of API Mesh for Adobe Developer App Builder

Reviewer: ```suggestion
description: Learn about the features of API Mesh for Adobe Developer App Builder.
```
